### PR TITLE
Add name in daily task

### PR DESCRIPTION
**Pull request by shivamchouhan123**

### CODE REVIEW

File: app/controllers/api/v1/daily_tasks_controller.rb [hunk 1]
```diff
@@ -10,18 +10,20 @@ class DailyTasksController < ApplicationController
 
       def index 
         if current_employee.is_hr? 
-          @daily_task_data= DailyTask.all.order(created_at: :desc)   
+          @daily_task_data= DailyTask.all.order(id: :desc)
```
Comment (ror-mahesh) [inline app/controllers/api/v1/daily_tasks_controller.rb:13]: Why you use instance variable here and also why you change order from created_at to ID??

**Review by ror-mahesh — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/daily_tasks_controller.rb (hunk 1 above)
Comment (shivamchouhan123) [reply app/controllers/api/v1/daily_tasks_controller.rb:13]: i am use id becuase employee data show in descending order

Comment (shivamchouhan123) [reply app/controllers/api/v1/daily_tasks_controller.rb:13]: data is not systomatically show then use in id on proper show data

Comment (ror-mahesh) [reply app/controllers/api/v1/daily_tasks_controller.rb:13]: okk,
SO why you use instance variable?

**Review by ror-mahesh — APPROVED** [review]
LGTM

**Pull request merged**